ODIN_II/SRC/ast_loop_unroll.cpp

```diff
@@ -599,109 +393,31 @@ ast_node_t* dup_and_fill_body(ast_node_t *ast_module, ast_node_t* body, ast_node
 			} 
 			else if(child->type == MODULE_INSTANCE && child->children[0]->type != MODULE_INSTANCE)
```

Comment:
set a boolean (i.e. unrolled) when this condition is met and then use the boolean for the next time you need to check that it was not unrolled.